spec/setup/helpers/perf_helper_spec.rb

```diff
@@ -41,13 +75,217 @@ def options
       hosts[0]['platform'] = Beaker::Platform.new('el-6-x86_64')
       allow(subject).to receive(:hosts).and_return(hosts)
       allow(subject).to receive(:agents).and_return(hosts)
-      expect(subject.hosts[0]).to receive(:install_package).with("https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm", '', nil, :acceptable_exit_codes => [0,1])
+      expect(subject.hosts[0]).to receive(:install_package).with("https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm", '', nil, :acceptable_exit_codes => [0, 1])
       allow(subject).to receive(:on)
       expect(subject).to receive(:on).with(hosts[0], 'chkconfig iptables off').once
       subject.install_epel_packages
     end
 
   end
 
+  context '.setup_r10k' do
+
+    # TODO: is this the best way to handle masters?
```

Comment:
I see no problem with this. I don't think we need to be fancy. 
We could test a list of masters with various lists of machines, some of which have master role and some that have compile_master, but then we're crossing the line into testing beaker methods rather than what we care about in our own methods.